Dockerfile

```diff
@@ -15,5 +15,8 @@ WORKDIR /opt/swoop-go
 # copy only swoop binary to container image
 COPY --from=APP /opt/swoop-go/swoop /opt/swoop-go/
 
+# copy fixtures to container image
```

Comment:
My only thing would be adding a comment that says why we are doing this, so we don't wonder the future and will know when we can remove this.

Reply:
done